src/scripts/Tooltip.jsx

```diff
@@ -449,7 +457,7 @@ export default class JoyrideTooltip extends React.Component {
 
     const overlayStyles = {
       cursor: disableOverlay ? 'default' : 'pointer',
-      height: document.body.clientHeight,
+      height: document.body.scrollHeight,
```

Comment:
**Required**

Surprised `clientHeight` was here, as its not all-browser encompassing. Assuming you want the entire scrollable document covered, you should leverage the util method `getDocHeight()` which uses the standard cross-browser way of calculating the scrollHeight.